### DISCUSSION TITLE
Docker mentioned in https://github.com/Akkudoktor-EOS/EOS

### DISCUSSION THREAD

**Discussion by briearn** [Akkudoktor-EOS/EOS#213] [Q&A]
Hi chaps,

on the main page of this project (https://github.com/Akkudoktor-EOS/EOS) its stating:

---snip---
Docker

docker compose up --build
Configuration
---snap---

thats a bit weak to understand what this is good for and if you need it or not.

Can somebody enlighten me please?

**Comment by XHalamaX** [comment]
You don't need docker to run this project but it can make sense to use docker in some cases. If you don't know what docker is or when to use it then it's maybe better if you watch some videos on YouTube about this.

The project itself is not easy to use at this stage and the documention is also in progress. You can have a look at the topic I created  [here](https://github.com/Akkudoktor-EOS/EOS/discussions/248) maybe then you get an overview (pv forecast & price forecast can be done by EOS itself, so my description is a bit out of date at the moment)
